### ISSUE TITLE
ImportError: cannot import name _unicode

### ISSUE THREAD

**Issue by amibiz** [getslash/slash#499]
trying out _slash_ like so: `$ slash run .`

causes the following error:
```
(.venv)slash » slash run .
1 tests collected...Session error caught -- ImportError: cannot import name _unicode

1 tests collected
ERROR: Could not load './.venv/lib/python2.7/genericpath.py' (/path/to/slash/.venv/lib/python2.7/posixpath.py:19 - cannot import name _unicode)
```

the only file in the current directory is called _testfile.py_ with one test case.
slash was installed on a freshly created virtuelenv, with pip v1.5.6 and python v2.7.10

**Comment by vmalloc** [comment]
Yes, Slash currently traverses *all* directories, and does not skip ones beginning with dots. I'll add a skip for such directories but it will only make it to the upcoming release (1.2.0), is that ok?

**Comment by vmalloc** [comment]
On second thought I think this will be better off in a hotfix release. Will get to it soon.

**Comment by amibiz** [comment]
Cool, thanks.

Thinking back on how I got to using the _dot_ in the first place, Its because I invoked slash with only the *run* command, like so: `$ slash run`
Currently, the default behavior is to print the _usage_ and and error indicating that no tests where specified:
```
(.venv)slash » slash run 
usage: slash run [-h] [-i] [-o PATH=VALUE] [--cov PARAM] [--cov-report PARAM]
                 [--cov-config PARAM] [--cov-append] [--xunit-filename PARAM]
                 [-f FILENAME] [--repeat-each NUM_TIMES] [-x]
                 [--repeat-all NUM_TIMES] [-k FILTER] [--tb PARAM]
                 [--truncate-console-lines yes/no] [-l DIR] [-q] [-v] [--pdb]
slash run: error: No tests specified
```
What do you think about changing the behavior so that when running the *run* command with no parameters will collect all tests under the current directory, while skipping _dot_ directories (so that slash won't traverse `.venv`or similar)?

**Comment by vmalloc** [comment]
Skipping the dot directories is a good idea in any case. As for the default tests location, it only makes sense for some projects, so I don't think we can make it the default.

However it can be easily customized using the following `.slashrc` in your directory root:

```python
from slash import config
config.root.run.default_sources = ['.']
```

**Comment by amibiz** [comment]
Using the above in my _.slashrc_ is a sound solution once the ImportError is fixed.
10x :)

**Comment by vmalloc** [comment]
Fixed in 1.1.4

**Comment by vmalloc** [comment]
Having issue with one of the tests, which is why 1.1.4 wasn't released. Will fix it and then release 1.1.5.
